clients/typescript/src/config/index.ts

```diff
@@ -44,16 +44,22 @@ export interface ElectricConfig {
   connectionBackOffOptions?: ConnectionBackOffOptions
 }
 
+export type ElectricConfigWithDialect = ElectricConfig & {
+  dialect?: 'SQLite' | 'Postgres' // defaults to SQLite
```

Comment:
nitpick: use JSDoc comments in cases like this, helps with hover tooltips


```suggestion
  /** defaults to SQLite */
  dialect?: 'SQLite' | 'Postgres'
```